spotinst/commons/common_organization_programmatic_user.go

```diff
@@ -0,0 +1,117 @@
+package commons
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/spotinst/spotinst-sdk-go/service/organization"
+	"log"
+)
+
+const (
+	OrgProgUserResourceName ResourceName = "spotinst_organization_programmatic_user"
```

Comment:
expand programmatic wherever necessary, not to use prog

Reply:
changed